internal/storage/vector_chunk_manager_test.go

```diff
@@ -0,0 +1,190 @@
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under the License.
+
+package storage
+
+import (
+	"context"
+	"os"
+	"path"
+	"strconv"
+	"testing"
+
+	miniokv "github.com/milvus-io/milvus/internal/kv/minio"
+	"github.com/milvus-io/milvus/internal/proto/etcdpb"
+	"github.com/milvus-io/milvus/internal/proto/schemapb"
+	"github.com/milvus-io/milvus/internal/util/paramtable"
+	"github.com/stretchr/testify/assert"
+)
+
+var Params paramtable.BaseTable
+
+func TestVectorChunkManager(t *testing.T) {
+	Params.Init()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	bucketName := "fantastic-tech-test"
+	minIOKV, err := newMinIOKVClient(ctx, bucketName)
+	assert.Nil(t, err)
+	defer minIOKV.RemoveWithPrefix("")
+
+	rcm := NewMinioChunkManager(minIOKV)
+
+	localPath := "/tmp/milvus/data"
+
+	lcm := NewLocalChunkManager(localPath)
+
+	schema := initSchema()
+	vcm := NewVectorChunkManager(lcm, rcm, schema)
+	assert.NotNil(t, vcm)
+
+	binlogs := initBinlogFile(schema)
+	assert.NotNil(t, binlogs)
+	for _, binlog := range binlogs {
+		rcm.Write(binlog.Key, binlog.Value)
+	}
+
+	content, err := vcm.ReadAll("108")
+	assert.Nil(t, err)
+	assert.Equal(t, []byte{0, 255}, content)
+
+	content, err = vcm.ReadAll("109")
+	assert.Nil(t, err)
+
+	floatResult := make([]float32, 0)
+	for i := 0; i < len(content)/4; i++ {
+		singleData := ByteToFloat32(content[i*4 : i*4+4])
+		floatResult = append(floatResult, singleData)
+	}
+	assert.Equal(t, []float32{0, 1, 2, 3, 4, 5, 6, 7, 0, 111, 222, 333, 444, 555, 777, 666}, floatResult)
+
+	content = make([]byte, 8*4)
+	byteLen, err := vcm.ReadAt("109", content, 8*4)
+	assert.Nil(t, err)
+	assert.Equal(t, 32, byteLen)
+
+	floatResult = make([]float32, 0)
+	for i := 0; i < len(content)/4; i++ {
+		singleData := ByteToFloat32(content[i*4 : i*4+4])
+		floatResult = append(floatResult, singleData)
+	}
+	assert.Equal(t, []float32{0, 111, 222, 333, 444, 555, 777, 666}, floatResult)
+
+	os.Remove(path.Join(localPath, "108"))
+	os.Remove(path.Join(localPath, "109"))
+}
+
+func newMinIOKVClient(ctx context.Context, bucketName string) (*miniokv.MinIOKV, error) {
+	endPoint, _ := Params.Load("_MinioAddress")
+	accessKeyID, _ := Params.Load("minio.accessKeyID")
+	secretAccessKey, _ := Params.Load("minio.secretAccessKey")
+	useSSLStr, _ := Params.Load("minio.useSSL")
+	useSSL, _ := strconv.ParseBool(useSSLStr)
+	option := &miniokv.Option{
+		Address:           endPoint,
+		AccessKeyID:       accessKeyID,
+		SecretAccessKeyID: secretAccessKey,
+		UseSSL:            useSSL,
+		BucketName:        bucketName,
+		CreateBucket:      true,
+	}
+	client, err := miniokv.NewMinIOKV(ctx, option)
+	return client, err
+}
+
+func initSchema() *etcdpb.CollectionMeta {
+	schema := &etcdpb.CollectionMeta{
+		ID:            1,
```

Comment:
You'd better not use number directly for "ID", "FieldID" ...
Use const instead.
You can refer to data_codec_test.go

Reply:
We don't have to and shouldn't do it.
When user use milvus, the ID of the collection is not necessarily 1, and the BoolFieldID is not necessarily 100. Only the TimestampField and RowIDField are const.
Changing these values into constants in ut will cause a lot of confusion when other coders want to know how to use data_codec through ut.
If we must use it, it should also be lowercase, otherwise it will be considered exposed.